pkgs/applications/audio/mopidy/default.nix

```diff
@@ -1,46 +1,39 @@
-{ newScope, python }:
+{ lib, pkgs, python }:
```

Comment:
Why change this function signature?

Reply:
I changed it to match other uses of makeScope/makeScope (i found 12) -- if it's an issue I can change it back.

Reply:
Using pkgs here is not required.

Reply:
```suggestion
{ lib, newScope, python }:
```

Reply:
Thanks, should be sorted now.